### ISSUE TITLE
[Bug] 200m hp xp takes screenshot every time i eat 

### ISSUE THREAD

**Issue by runeliteissuesmaker** [Nightfirecat/plugin-hub-plugins#8]
![image](https://user-images.githubusercontent.com/71134694/97395730-cb544c00-18a2-11eb-9e4a-06dc25d1ea73.png)

Every time i eat a food it gives me a pop up and takes a screenshot

**Comment by Nightfirecat** [comment]
Oh, that's not good. I do know why this happens--will have a fix out soon.

**Comment by Nightfirecat** [comment]
This is fixed in 1d98172d66f33fa17e6a2bee63c06da4cc3b6f04. I will close this issue once it is merged into plugin-hub.
